snekbox/nsjail.py

```diff
@@ -163,37 +193,49 @@ def python3(
                 "--",
                 self.config.exec_bin.path,
                 *self.config.exec_bin.arg,
-                *py_args,
-                code,
-            )
+                # Filter out empty strings (causes issues with python cli)
```

Comment:
Only empty strings at the start should be removed. Empty strings elsewhere could conceivably be valid arguments for the script.